### PR TITLE
deleted methods

### PR DESCRIPTION
**Pull request by katerynaklymina**

### CODE REVIEW

File: src/main/java/page_steps/selenium/ItemDetailsPageSteps.java [hunk 1]
```diff
@@ -12,19 +12,6 @@
...
-    }
-
-    public boolean isBuyInOneClickEnabled() {
-        return itemDetailsPage.getBuyInOneClick().isEnabled();
-    }
-
-    public boolean isBuyInInstallmentEnabled() {
-        return itemDetailsPage.getBuyInInstallment().isEnabled();
-    }
-
```
Comment (ketrin111) [inline src/main/java/page_steps/selenium/ItemDetailsPageSteps.java:27]: Why did you delete this?

Comment (katerynaklymina) [reply src/main/java/page_steps/selenium/ItemDetailsPageSteps.java:27]: They are not usable
